src/cachehash/main.py

```diff
@@ -1,6 +1,6 @@
 import json
 import sqlite3
-
+import os
```

Comment:
In the future, please keep imports in alphabetical order unless they're of the from x import y style per [PEP 8](https://peps.python.org/pep-0008/#imports)